src/components/RouteWrapper.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { ReactElement, ReactNode } from 'react'
+import { StatusBar, StatusBarStyle } from 'react-native'
+import styled from 'styled-components/native'
+
+import theme from '../constants/theme'
+
+interface Props {
+  backgroundColor?: string
+  barStyle?: StatusBarStyle
```

Comment:
How about making this a boolean, i.e:
```suggestion
  lightStatusBarContent?: boolean
```
Then you can use a default of `false` and it should be easier to use and it is always clear what the default is.